bin/update_schedule.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash#
-
+ACTIVE_COHORT=6
+NEXT_COHORT=$((ACTIVE_COHORT+1))
+EXPORT_URL="https://docs.google.com/spreadsheets/d/1jXJH8J4MKee80ODcgWoxpvPkV8ob0tw86Fov8nlXxXw/export?format=csv&gid=1181436982"
```

Comment:
The issue is that the URL changes for each cohort. It would be maybe better to store the different links in a list or something similar

Reply:
They are all in this bash script